ui/src/components/setlist/SetlistListContainer.tsx

```diff
@@ -105,7 +112,7 @@ const SetlistListContainer: FC = (): ReactElement => {
 
   useEffect(() => {
     getSetlistsAndFolders();
-  }, [getSetlistsAndFolders]);
+  }, [openDrawer]);
```

Comment:
[important]: same issue here with the `getPeople()`, but I think this time better to call the function when `openDrawer` is changed so we can fast reload when something is changed in the drawer.